### ISSUE TITLE
[BUG] Whenever I download bloxstrap it opens up roblox

### ISSUE THREAD

**Issue by Cumpanasu1** [pizzaboxer/bloxstrap#2306]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.

### What problem did you encounter?

I had bloxstrap before but i uninstalled it then after i tried to install it again it just opens up roblox. I used the discord server to get some help but i still couldn't fix this problem https://discord.com/channels/1099468797410283540/1260535221347029023.

**Comment by nakoyasha** [comment]
Are you sure you're opening "Bloxstrap Menu" and not "Play Roblox"? The latter will always open the app (or the website if it's disabled, i think?)

**Comment by pizzaboxer** [comment]
> (or the website if it's disabled, i think?)

Nope, not anymore

**Comment by Cumpanasu1** [comment]
> Are you sure you're opening "Bloxstrap Menu" and not "Play Roblox"? The latter will always open the app (or the website if it's disabled, i think?)

i dont even have the menu, i clicked the link where you download bloxstrap and after i open the folder it opens up roblox for me.

**Comment by pizzaboxer** [comment]
> i dont even have the menu, i clicked the link where you download bloxstrap and after i open the folder it opens up roblox for me.

https://github.com/pizzaboxer/bloxstrap/wiki/Opening-the-Bloxstrap-menu ?

**Comment by Cumpanasu1** [comment]
> > i dont even have the menu, i clicked the link where you download bloxstrap and after i open the folder it opens up roblox for me.
> 
> https://github.com/pizzaboxer/bloxstrap/wiki/Opening-the-Bloxstrap-menu ?
Yes i looked in my starting menu for it and i still cloudnt find it. Not even in the control pannel at the uninstall apps section

**Comment by nakoyasha** [comment]
Sounds like a corrupted install - please send your logs! (%localappdata%/Bloxstrap/Logs)

**Comment by Cumpanasu1** [comment]
> Sounds like a corrupted install - please send your logs! (%localappdata%/Bloxstrap/Logs)

![image](https://github.com/user-attachments/assets/5f1c6574-97ae-4cda-ab19-5182c8fd636f)
here

**Comment by Cumpanasu1** [comment]
maybe this is also usefull 
![image](https://github.com/user-attachments/assets/77dc0257-d73e-4506-a0b9-36404838f075)

**Comment by pizzaboxer** [comment]
Where is that menu shortcut located? Right-click it and click "Open file location"

**Comment by Cumpanasu1** [comment]
![image](https://github.com/user-attachments/assets/73b479b8-8ab5-4a97-a67e-a07cc2e36420)

**Comment by pizzaboxer** [comment]
No, I'm talking about the file you selected, this one:

![image](https://github.com/user-attachments/assets/2089306f-c0e9-4a79-9d76-7c4d019381c7)

Where is it located?

**Comment by Cumpanasu1** [comment]
I pressed on it then right clicked and it then open file location and it appears as bloxstrap exe

**Comment by Cumpanasu1** [comment]
should i do a video proof?

**Comment by pizzaboxer** [comment]
Oh, I forget that Windows makes this really annoying.

Right-click on the shortcut, click "Properties", go to the General tab, and copy and paste the whole location field here.

**Comment by Cumpanasu1** [comment]
C:\VTRoot\HarddiskVolume4\Users\Comorasu\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Bloxstrap

**Comment by pizzaboxer** [comment]
What is wrong with your computer? This doesn't make any sense.

Just so I can double check, if you go to %localappdata%, what path does it resolve to?

**Comment by Cumpanasu1** [comment]
so if i press win + r and then type %localappdata% and i search for bloxstrap i find an empty folder 
![image](https://github.com/user-attachments/assets/bdf8f868-5a80-4f66-a5b8-11b6f7e51575)
![image](https://github.com/user-attachments/assets/3f322598-7682-40b0-9e21-333a309d3a24)

**Comment by pizzaboxer** [comment]
Oh, this is an antivirus thing. More specifically, Comodo's sandbox feature. Add an exception for it, reinstall Bloxstrap, and you'll be able to use it normally.
